flink-streaming-connectors/flink-connector-kafka-0.8/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/Kafka08Fetcher.java

```diff
@@ -78,7 +79,10 @@
 	private final long invalidOffsetBehavior;
 
 	/** The interval in which to automatically commit (-1 if deactivated) */
-	private final long autoCommitInterval; 
+	private final long autoCommitInterval;
+
+	/** The metric group of this operator */
+	private final MetricGroup metricGroup;
```

Comment:
the 0.8 Fetcher has a `MetricGroup` field, while the 0.9 Fetcher only creates a local variable inside run.  Let's make this more consistent.